static-site/content/docs/05-contributing/01-sharing-data.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "How to share your analyses"
+---
+
+### Our Philosophy: Facilitating Open-Science & Sharing of Results
+
+From the beginning, Nextstrain has focused on open, real-time, pre-publication sharing of results.
+Every situation is different and over time we've tried to develop a range of different approaches to take steps in this direction wherever possible.
+
+Here is a summary of the different ways one can share data through [nextstrain.org](/) or using the tools which are behind nextstrain:
+
+<br/>
+
+| Name                   | Data lives where?           |  Accessed via             |  Access            |
+| ---------------------- | --------------------------- | ------------------------- | ------------------ |
+| [Nextstrain Community] | Your own github repo        | nextstrain.org/community/ | Public             |
+| Nextstrain Groups      | S3 bucket which you manage  | nextstrain.org/groups/... | Public or private  |
+| [auspice.us]           | On your computer            | [auspice.us]              | Private            |
+| [Custom auspice server]| Wherever you choose         | your own server's URL     | Your choice        |
+
+We are always excited to help you contribute to Nextstrain, no matter what shape this takes.
+Please [get in touch with us](mailto:hello@nextstrain.org) with any specific questions and we'll be happy to help.
+
+
+### Centrally Maintained, Regularly Updated Datasets
+
+There are a number of datasets which are run by the Nextstrain team or close collaborators, for instance [SARS-CoV-2](/ncov/global), [seasonal influenza](/flu), and [West Nile virus](/WNV/NA).
+Each individual dataset contains links to the scientists who maintain it.
+
+We're exited to collaborate with more research groups and expand these datasets -- please [get in touch with us](mailto:hello@nextstrain.org) if this is something you'd like to be involved in.
+
+
+### Community Maintained Datasets
+
+To make supporting this philosophy as easy as possible, we have created a "community builds" functionality, whereby GitHub is used to store the results of your analyses and the results are available for everyone via nextstrain.org.
+This is behind the [community builds](/#community) which you can see on the main page.
+
+See [this page](community-builds) for more information, including a step-by-step guide on how you can get your datasets up as a community build.
+
+### Nextrain Groups
+
+Groups are an initiative to allow research labs, public health entities and others to manage their own datasets and narratives and share them through nextstrain.org.
+Groups can either be private or public in order to allow data sharing to the correct audience -- you can see an example of a public group [here](nextstrain.org/groups/blab/).
```

Comment:
The blab groups page link didn't work for me. Replacing it with `https://nextstrain.org/groups/blab` seemed to fix it!

Reply:
Thanks! Now fixed